infra/base-images/base-builder-fuzzbench/fuzzbench_run_fuzzer

```diff
@@ -25,7 +25,7 @@ mkdir $SEED_CORPUS_DIR
 rm -rf $OUTPUT_CORPUS_DIR
 mkdir $OUTPUT_CORPUS_DIR
 export FUZZER=$FUZZING_ENGINE
-export SNAPSHOT_PERIOD=900
+export SNAPSHOT_PERIOD=$((MAX_TOTAL_TIME / 3))
```

Comment:
Maybe comment this is needed for fuzzbench but we don't actually use it.

Reply:
Actually, might be worth checking in fuzzbench's code for running fuzzers if we can skip corpus snapshotting.